### PR TITLE
#112078867 As a user, I can see a list of postings that need to be completed today.

### PR DESCRIPTION
**Pull request by matt-wratt**

### CODE REVIEW

File: src/style/variables.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@ $cco-bright-green: #72BF44;
 $background-color: white;
 
 // Misc colors:
-$light-grey: #c2c2c1;
+//$light-grey: #c2c2c1;
```
Comment (lukemcfarlane) [inline src/style/variables.scss]: Dead code.

File: src/style/components/postings.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    font-family: Proxima Nova Alt Regular;
+
+    &:first-child {
+      margin-top: 18.8px;
+    }
+
+    .row {
+      margin-left: 40px;
+
+      strong {
```
Comment (lukemcfarlane) [inline src/style/components/postings.scss]: Could this be moved to `base.scss` so that we can reuse?

**Comment by lukemcfarlane** [comment]
Suggest using rems instead of px.

File: src/actions/postings.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { apiFetch, async, runAsync, wrapAsync } from '../utils'
+
+export const REQUEST_POSTINGS = 'REQUEST_POSTINGS'
```
Comment (lukemcfarlane) [inline src/actions/postings.js]: Is there any way that we could make these consts more enum-like?

**Comment by lukemcfarlane** [comment]
Would be good to add a `.env.example`.

File: src/actions/postings.js (hunk 3 above)
Comment (matt-wratt) [inline src/actions/postings.js]: yes... and no, we can change the approach if this becomes too tedious.

**Pull request merged**